portfolio/src/main/webapp/index.html

```diff
@@ -29,114 +37,123 @@ <h5 class="subtitle">
             self-improvement. 4. dreams big and gets s#$% done
           </em>
         </h5>
-        <p class="main">
-          <em>"Vulnerability doesn't come after trust - it precedes it."</em> ~
-          <a
-            class="list-link"
-            href="https://fourminutebooks.com/the-culture-code-summary/"
-            target="#"
-            ><u>The Culture Code:</u></a
-          >
-        </p>
         <ul class="main">
           <li>
-            '14: Barely made the basketball team and got bullied for my weight
-          </li>
-          <li>
-            '14-'15: Lost over 50 pounds and became a starter for the same team
+            I was born and raised in Calgary, Alberta
           </li>
           <li>
-            '16-'17: Became a starter for my high school and AAU basketball
-            teams
+            I have an 
+            <a class="list-link" 
+              href="images/family.png" data-lightbox="Family" data-title"Family">
```

Comment:
missing =

Reply:
Applies to other data-title's in this file

Reply:
Got it.